ui-core/src/styles/inputs/slider.css

```diff
@@ -0,0 +1,72 @@
+.range-wrapper {
+  margin: 50px;
+  width: 80%;
+  height: 10px;
+
+  .range-slider {
+    -webkit-appearance: none;
+    appearance: none;
+    height: 10px;
+    width: 100%;
+  
+    &::-webkit-slider-runnable-track {
+      -webkit-appearance: none;
+      appearance: none;
+      height: 4px;
+      border-radius: 2px;
+      box-shadow: 0px 0px 0px 0.5px rgba(0, 0, 0, 0.25) inset;
+      background-color: rgba(0, 0, 0, 0.05);
+    }
+    
+    &::-moz-range-track {
+      -webkit-appearance: none;
+      appearance: none;
+      height: 4px;
+      border-radius: 2px;
+      box-shadow: 0px 0px 0px 0.5px rgba(0, 0, 0, 0.25) inset;
+      background-color: rgba(0, 0, 0, 0.05);
+    }
+    
+    &::-webkit-slider-thumb {
+      -webkit-appearance: none;
+      appearance: none;
+      width: 10px;
+      height: 10px;
+      margin-top: -3px;
+      border-radius: 50%;
+      border: 0.5px solid rgba(73, 70, 65, 1);
+      box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.25);
+      opacity: 1;
+      background-color: rgba(255, 255, 255, 1);
+    }
+    
+    &::-moz-range-thumb {
+      -webkit-appearance: none;
+      appearance: none;
+      width: 10px;
+      height: 10px;
+      margin-top: -3px;
+      border-radius: 50%;
+      border: 0.5px solid rgba(73, 70, 65, 1);
+      box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.25);
+      opacity: 1;
+      background-color: rgba(255, 255, 255, 1);
+    }
+  }
+
+  .range-slider-disabled {
```

Comment:
Behavior is nice but @thibautsailly as there is no example in the mockup, could you tell us if it is what you want ?

![image](https://github.com/user-attachments/assets/a9da8f26-e9fa-47a9-9008-e5803c9e9fdf)